Dockerfile

```diff
@@ -422,6 +422,8 @@ RUN pip install flashtext && \
     pip install easyocr && \
     # Keep JAX version in sync with GPU image.
     pip install jax==0.2.12 jaxlib==0.1.64 && \
+    # ipympl adds interactive widget support for matplotlib
+    pip install ipympl && \
```

Comment:
Probably hard to add a test for this right?

Reply:
Yeah I don't think theres a way to test an interactive widget like this :(

Reply:
Adding a version pin to make sure it stays compatible